api/cloud.proxy.js

```diff
@@ -43,6 +47,7 @@ function IoTKitCloud(logger, deviceId, customProxy) {
     me.deviceId = deviceId;
     me.deviceName = deviceConf.device_name;
     me.gatewayId = deviceConf.gateway_id || deviceId;
+    me.store = Sensor.init("device.json", me.logger);
```

Comment:
Hopefully not. It initializes in-memory property with sensor list from file or empty array.